Changes in webpage

AtharvaParkale opened the pull request.

chehak123 commented:
Kindly add screenshots of the changes you have done and mention the issue you have solved.

AtharvaParkale commented:
> Kindly add screenshots of the changes you have done and mention the issue you have solved.

Okay

AtharvaParkale commented:
![GirlMeetup - Google Chrome 1_29_2021 6_19_18 PM](https://user-images.githubusercontent.com/71985594/106277474-35440f00-625f-11eb-893f-4e3d271874a9.png)

1. Changed content inside your website.
2. Replaced ipsum with my content.
